src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/Streaming/DataStreamReaderTests.cs

```diff
@@ -63,6 +62,7 @@ public void TestSignaturesV2_3_X()
                 dsr.Parquet(Path.Combine(TestEnvironment.ResourceDirectory, "users.parquet")));
             Assert.IsType<DataFrame>
                 (dsr.Text(Path.Combine(TestEnvironment.ResourceDirectory, "people.txt")));
+            Assert.IsType<DataFrame>(dsr.Format("json").Option("path", jsonFilePath).Load());
```

Comment:
Setting `path` option and then calling `load` is not supported by default unless  `spark.sql.legacy.pathOptionBehavior.enabled` conf is set.  Reverse order in test.

https://github.com/apache/spark/blob/bb9abb098a1e26c220546616d655380c479b0e42/sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala#L259